test.py

```diff
@@ -85,13 +85,14 @@ def test_user(self):
 
     def test_search(self):
         """test search result"""
-        result = self.loklak.search('doctor who')
+        result = self.loklak.search('doctor who', None, None, None, 18)
         self.assertTrue('error' in self.loklak.search())
         self.assertTrue('statuses' in result)
         self.assertTrue(isinstance(result['statuses'], list))
         self.assertTrue(len(result['statuses']) >= 1)
         self.assertEqual(len(result['statuses']),
                          int(result['search_metadata']['count']))
+        self.assertEqual(result['search_metadata']['itemsPerPage'], '18')
```

Comment:
It'd be better if this was for `self.assertEqal(result['search_metadata']['count'], 18)`